# Header <stdint.h> only included if HAVE_STEDINT_H macro is set
# May 8th 2018 by B. Hajgato (Free University Brussels - VUB)
--- libmo_unpack-3.1.2/tests/CMakeLists.txt.orig	2015-10-09 12:44:17.000000000 +0200
+++ libmo_unpack-3.1.2/tests/CMakeLists.txt	2018-05-08 15:03:00.115085372 +0200
@@ -6,3 +6,4 @@
 add_executable(test_rle check_rle.c)
 target_link_libraries(test_rle ${LIBS})
 add_test(test_rle ${CMAKE_CURRENT_BINARY_DIR}/test_rle)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DHAVE_STDINT_H")
